app/javascript/shared/components/ResizableTextArea.vue

```diff
@@ -162,3 +170,8 @@ export default {
   },
 };
 </script>
+<style scoped>
+textarea {
+  height: inherit;
```

Comment:
I'm not using tailwindcss as we have to use h-[inherit] which is getting overridden by lots of our existing styles.